tests/functional/_intern.js

```diff
@@ -59,7 +59,7 @@ intern.configure({
   reporters: [args.reporters ? args.reporters : "pretty"],
 
   functionalSuites: [
-    args.functionalSuites ? args.functionalSuites : "./tests/functional/*.js"
+    args.functionalSuites ? args.functionalSuites : "./tests/functional/search-auth.js"
```

Comment:
(tip: when running a single test, you can do the following:

`node ./tests/functional/_intern.js functionalSuites=tests/functional/search-auth.js` 

And then it will only run that test)

Reply:
Not useful to you now, but I'm going to fix some things in https://github.com/webcompat/webcompat.com/issues/2193 so you can just do `npm run:test:js -- --functionalSuites=blah --grep=interaction`, then you don't have to comment anything out either.

Reply:
👍